lib/index.js

```diff
@@ -2,9 +2,13 @@
 
 var CharsetLib = require('./charset');
 var EncodingLib = require('./encoding');
+var LanguageLib = require('./language');
 
 exports.charset = CharsetLib.charset;
 exports.charsets = CharsetLib.charsets;
 
 exports.encoding = EncodingLib.encoding;
 exports.encodings = EncodingLib.encodings;
+
+exports.language = LanguageLib.language;
+exports.languages = LanguageLib.languages;
```

Comment:
If I don't I'm going to have lines like this:
``` exports.language = Language.language; ```

Seems like a differentiator or some sort is a bonus for readability.  It could be something else, but "lib" fits the bill and is short.